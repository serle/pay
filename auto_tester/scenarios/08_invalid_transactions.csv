type,client,tx,amount
deposit,1,1,100.0
invalid,1,2,50.0
withdrawal,1,3,25.0
dispute,999,999,
resolve,999,999,
