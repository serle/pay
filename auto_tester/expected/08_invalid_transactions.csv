client,available,held,total,locked
1,75.0000,0.0000,75.0000,false
